ST/31_sending_and_validating_API_requests_and_configuration_of_API_client/APITestInCSharp/README.adoc

```diff
@@ -0,0 +1,42 @@
+= API Client Configuration
```

Comment:
Hello
Please rename the file as per the covered topic and delete README from the name  :)

Reply:
Hello, done, README changed to HowToRun file) And it is extra materials, and not the file with the covered topic